### PR TITLE
[plugin] pymongo compatible with mongodb cluster

### PR DESCRIPTION
**Pull request by cooolr**
replace the `this.collection.database.client.address` with the `this.collection.database.client.nodes` to be compatible with the mongodb cluster

### CODE REVIEW

File: skywalking/plugins/sw_pymongo.py [hunk 1]
```diff
@@ -101,8 +101,8 @@ def inject_bulk_write(_Bulk, bulk_op_map):
     _execute = _Bulk.execute
 
     def _sw_execute(this: _Bulk, *args, **kwargs):
-        address = this.collection.database.client.address
-        peer = "%s:%s" % address
+        nodes = this.collection.database.client.nodes
```
Comment (kezhenxu94) [inline skywalking/plugins/sw_pymongo.py:104]: So when the MongoDB is in single-node mode, the `this.collection.database.client.nodes` will also contain one element?

**Review by kezhenxu94 — APPROVED** [review]
@cooolr thank you!

**Review by Humbertzhang — APPROVED** [review]
LGTM

File: skywalking/plugins/sw_pymongo.py (hunk 1 above)
Comment (Humbertzhang) [reply skywalking/plugins/sw_pymongo.py:104]: I looked at it and the answer is yes.

**Pull request merged**